src/main/java/nextstep/subway/domain/Section.java

```diff
@@ -84,18 +82,17 @@ public Station getDownStation() {
     }
 
     public void updateDownStation(Section section) {
-        if(section.getDistance()>=this.distance){
-            throw new IllegalArgumentException(ErrMsg.DISTANCE_TOO_LONG);
-        }
         this.downStation = section.getUpStation();
-        this.distance -= section.getDistance();
+        this.distance = this.distance.subtract(section.getDistance());
```

Comment:
```suggestion
        this.distance = section.subtractDistance(distance);
```

객체지향적 관점을 고려해보면, `getter`를 줄일수 있겠네요!

Reply:
오 그렇군요! 수정했습니다.